charts/victoria-metrics-cluster/templates/vminsert-deployment.yaml

```diff
@@ -110,20 +110,28 @@ spec:
               containerPort: {{ include "split-host-port" .Values.vminsert.extraArgs.opentsdbListenAddr }}
           {{- end }}
           readinessProbe:
+            {{- if not .Values.vminsert.probe.readiness.extra }}
```

Comment:
1. I think If those are extra readiness/liveness, they shouldn't override the default one;
2. Also need to add corresponding values in https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-cluster/values.yaml#L72